connection_testcases.json

```diff
@@ -998,7 +998,6 @@
       "waitUntilAvailable": "PT30S"
     },
     "warnings": [
-      "docker_tcp_port",
```

Comment:
This file is generated now. You have to find this test in the `tests/` directory and update it there. Then run `./tools/rebuild.ts` to update the connection_testcases.json file.

Reply:
oooh thank u!